packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -78,7013 +73,9090 @@
       }
-      /*if both has the same sign*/
-      else if (this->signbit == rhs.signbit){
-
-        if (this->val < rhs.val){//if my value is smaller,
-          return this->signbit;//then if we both are positive return true.
-                              //if we both are negative, return false.
-        }
-        else if (this->val > rhs.val){//if my value is larger,
-          return !this->signbit; //then if we both are positive return false.
-                                //if we both are negative, return true.
-        }
-        else { //if both are equal.
-          return false;
-        }
+      else if(this->val > rhs.val) {//if my value is larger,
+        return !this->signbit; //then if we both are positive return false.
+                              //if we both are negative, return true.
       }
-      else {
-        /*if I am positive, the other is negative*/
+      else { //if both are equal.
         return false;
       }
-
     }
-    bool operator>(const uSignedSortItem<IT, WT, SIGN>& rhs) const {
+    else {
       /*if I am positive, the other is negative*/
-      if (this->signbit > rhs.signbit){
-        return true;
+      return false;
+    }
+  }
+  bool operator>(const uSignedSortItem<IT, WT, SIGN>& rhs) const {
+    /*if I am positive, the other is negative*/
+    if(this->signbit > rhs.signbit) {
+      return true;
+    }
+    /*if both has the same sign*/
+    else if(this->signbit == rhs.signbit) {
+      if(this->val < rhs.val) {//if my value is smaller,
+        return !this->signbit;//then if we both are positive return false.
+                            //if we both are negative, return true.
       }
-      /*if both has the same sign*/
-      else if (this->signbit == rhs.signbit){
-
-        if (this->val < rhs.val){//if my value is smaller,
-          return !this->signbit;//then if we both are positive return false.
-                              //if we both are negative, return true.
-        }
-        else if (this->val > rhs.val){//if my value is larger,
-          return this->signbit; //then if we both are positive return true.
-                                //if we both are negative, return false.
-        }
-        else { // if they are equal
-          return false;
-        }
+      else if(this->val > rhs.val) {//if my value is larger,
+        return this->signbit; //then if we both are positive return true.
+                              //if we both are negative, return false.
       }
-      else {
-        /*if I am negative, the other is positive*/
+      else { // if they are equal
         return false;
       }
     }
-    bool operator<=(const uSignedSortItem<IT, WT, SIGN>& rhs){
-      return !(*this > rhs);}
-    bool operator>=(const uSignedSortItem<IT, WT, SIGN>& rhs){
-      return !(*this  < rhs);}
+    else {
+      /*if I am negative, the other is positive*/
+      return false;
+    }
+  }
+  bool operator<=(const uSignedSortItem<IT, WT, SIGN>& rhs) {
```

Comment:
If you use `std::sort`, you should only need `operator<`.

Reply:
@kddevin Recommended changes would simplify code a lot. But this code is not really changed with the PR, just cleaned up formatting, so we may want to consider as a separate PR.

Reply:
I removed operator> and logic now uses just operator< but did not switch to the std::sort - not sure exactly what uqsort does.